.gitignore

```diff
@@ -1,4 +1,5 @@
 # ThePlatform
+Secure.py
```

Comment:
What is this? There is no secure.py. We removed it in favor of secure/

Unless something is going on I'm not thinking about, remove this.